man/toTinytable.Rd

```diff
@@ -0,0 +1,46 @@
+\name{toTinytable}
+\alias{toTinytable}
+\title{
+Convert \code{tabular} object to \code{tinytable} format.
+}
+\description{
+Converts the output of the \code{\link{tabular}} and related
+functions to a format consistent with the output of the
+\code{\link[tinytable]{tt}} function, so that it can be 
+customized using the \pkg{tinytable} package.
+}
+\usage{
+toTinytable(table, ...)
+}
+\arguments{
+  \item{table}{
+An object of class \code{tabular}.
+}
+  \item{...}{
+Additional arguments to pass to \code{\link[tinytable]{tt}}.
+}
+}
+\value{
+An object of class \code{tinytable}, suitable for passing
+to functions in the \pkg{tinytable} package. These tables
+can be exported to several formats, including LaTeX, HTML,
+Markdown, Word, Typst, PDF, and PNG.
+}
+
+\seealso{
+\code{\link[tinytable]{tinytable-package}} 
+}
+\examples{
+if (requireNamespace("tinytable") && 
+    (!requireNamespace("pkgdown") || !pkgdown::in_pkgdown())) {
+  tab <-  tabular( (Species + 1) ~ (n=1) + Format(digits=2)*
+          (Sepal.Length + Sepal.Width)*(mean + sd), data=iris )
+  print(toTinytable(tab, theme = "striped"))
+  cat("\n")
+  
+  tab <- toTinytable(tab)
+  tab <- tinytable::style_tt(tab, i = 1:2, background = "teal", 
+    color = "white")
+  print(tab)
```

Comment:
It might look better to users to print to HTML in the example.

Reply:
`print.tinytable(tab)` tries to be "smart" about the default output format.

If `interactive()`:

* HTML in the Rstudio Viewer pane if available.
* Markdown otherwise 

If not `interactive()`:

* HTML if `knitr` is installed and `knitr::is_latex_output()`
* LaTeX if `knitr` is installed and `knitr::is_latex_output()`
* Markdown otherwise

We can force a format with `print(tab, "html")`, which will call `utils::browseURL()` on a temp file, but this is not 100% safe. For instance, it sometimes fails on my Ubuntu-on-Windows via Subsystem Linux (WSL). Defaulting to markdown in unknown environments seems safer to me.

But I'm happy to switch to `print(tab, "html")` for your documentation if you prefer.

Reply:
Let's leave it as is.